### ISSUE TITLE
error[E0793]: reference to packed field is unaligned -during anchor download

### ISSUE THREAD

**Issue by Sam-Berman1** [coral-xyz/anchor#2567]
cargo install --git https://github.com/coral-xyz/anchor avm --locked --force

Sometimes may return:

```
.cargo\registry\src\index.crates.io-6f17d22bba15001f\ntapi-0.3.7\src\ntexapi.rs:2783:52
     |
2783 |         *tick_count.QuadPart_mut() = read_volatile(&(*USER_SHARED_DATA).u.TickCountQuad);
     |                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |
     = note: packed structs are only aligned by one byte, and many modern architectures penalize unaligned field accesses
     = note: creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)
     = help: copy the field contents to a local variable, or replace the reference with a raw pointer and use `read_unaligned`/`write_unaligned` (loads and stores via `*p` must be properly aligned even when using raw pointers)
```

Changing the .cargo/git/checkouts/anchor/xxxxxxx/avm/Cargo.toml
To include:

tokio = { version = "1.28.0", features = ["full"] }

& 

.cargo/git/anchor/xxxxxx/cli/Cargo.toml

from "~1.14.0" to 1.28.0

Fixed this issue.
Thankful for https://stackoverflow.com/questions/76173195/my-rust-project-is-broken-all-of-the-sudden for hinting

**Comment by acheroncrypto** [comment]
```
cargo install --git https://github.com/coral-xyz/anchor avm --force
```

should pull the latest `tokio` version(without `--locked` argument).

 `avm` doesn't have a direct dependency to `tokio` but it's getting `tokio 1.14.1` because `solana-client` specifies `tokio ~1.14.1`. Looks like they've just bumped [`tokio` to `1.29.1`](https://github.com/solana-labs/solana/pull/32430) and [applied `ntapi` patch](https://github.com/solana-labs/solana/pull/32488) but that is not yet released.

**Comment by acheroncrypto** [comment]
This problem only seems to happen on Windows, you'll have much better experience if you use WSL for developing on Solana if you are on Windows.

**Comment by acheroncrypto** [comment]
This should be fixed in `0.29.0` https://github.com/coral-xyz/anchor/blob/fc3905a099eb6938bafe60f2396f821360e89ec1/Cargo.lock#L5009-L5010
